l10n/bundle.l10n.json

```diff
@@ -108,12 +105,6 @@
   "{0} does not contain any source files": "{0} does not contain any source files",
   "Workspace folder not specified": "Workspace folder not specified",
   "No source files selected to be migrated": "No source files selected to be migrated",
-  "Extension: {0}\n": "Extension: {0}\n",
```

Comment:
In the Code4i PR which I opened (and is still under review), I added APIs so that Project Explorer can render the same style of tooltips. These tooltips will still include all this information (including ASP) -> https://github.com/codefori/vscode-ibmi/pull/2122/files#diff-e1962db673dbf505e2c74deb034072677a2b4fca832d78a983eebeecac55df85R976